### DISCUSSION TITLE
Using the pump panel without a working WIFI

### DISCUSSION THREAD

**Discussion by mrjeth** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#522] [Q&A]
I have finally filled my Hollywood SPA (S100101) with water and tested the controller in real life. It works very well without issues since I started it a week ago. I have also the controller connected MQTT and HA. 

However, Yesterday I accidently switch off the power to the pump, but immediately turned it on again. Then I discovered that my WIFI also was down, in fact totally down and I need to get a new router.
It looks like the great WiFi-remote-for-Bestway-Lay-Z-SPA ill not start without a working WIFI!
Is there a way to start it without WIFI after a power down and only use the buttons? If not, I think it would be a good candidate for a future improvement.
Maybe I have missed something in the setup?

I am using is the Wemos d1 mini without the PCB and level converters. The cables and the connectors are connected directly to the controller, but I don’t think my problems has anything to do with it.
Thank you for a great project.
Tomas

**Comment by visualapproach** [comment]
Hi, great to hear it worked fine for you! I bet the autoportal has activated since the Wi-Fi is done. You can deactivate it from the web ui. It should then work without Wi-Fi. (From the panel)

**Reply by visualapproach** [reply]
I guess you have to arrange a temporary Wi-Fi from the phone, and enter the credentials in layzspa portal, then via the phone change the settings in web ui

**Reply by mrjeth** [reply]
So you say that I must first connect to the lazyspa's wifi AP and when I am logged in I must configure some settings. 
I am not sure what settings that is necessary, please advice. Or is it so that once you have logged in to the AP the buttons will be connected to the panel?

**Reply by visualapproach** [reply]
The AP is blocking the code. So you need to turn it off by getting to the web config page. You need Wi-Fi to get to that page.

So visit AP and set a new ssid+password.
Start internet sharing on your phone with same ssid+pwd.
Restart pump and check the IP on the display
On phone go to that IP
Find setting and save and reboot

**Reply by visualapproach** [reply]
Or go to the router store, it will be easier LoL 😂

**Comment by mrjeth** [comment]
I understand what is needed but to learn my wife would be a challenge, 

Maybe a candidate for improvement in a future release. If no Wifi network found in x minutes then fail back to simple mode.

Thank you anyway for your feedback and comments!

Best regards
Tomas

**Reply by visualapproach** [reply]
You're welcome! The feature is already there but you have to prepare it before you break the Wi-Fi. AP is needed at first startup. Then you may or may not want the auto AP to be active depending on your needs. It's awesome if you just want to change ssid on your Wi-Fi, but if you have outages when not supervised you should turn it off. Should probably be mentioned in the manual ...

**Reply by mrjeth** [reply]
Maybe I misunderstood you?  
You are saying that when the AP tick box is switched off,  the need for an active WIFI is no longer needed? I have off course already switch to another SSID and used it , but then after a while the network disappeared and I ended up in this situation. However i never ticked of the AP.

Thanks a lot!

**Reply by visualapproach** [reply]
Yes that's what I meant 👍🏼

**Comment by mrjeth** [comment]
Thank you for all help!
